public/assets/scss/_navbar.scss

```diff
@@ -0,0 +1,149 @@
+%heading-font-stack {
+    color: $blue;
+    font-size: 3rem;
+}
+%heading-text-font-stack {
+    color: $gray-700;
+    font-size: 1.2rem;
+    font-weight: 300;
+}
+%title-font-stack {
+    color: $gray-700;
+    font-size: 1.2rem;
+    font-weight: 400;
+}
+%button-stack {
+    color: $white;
+    background-color: $purple;
+    border-radius: 0.2rem;
+    padding: 0.5rem 2rem;
+    font-size: 1rem;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
Here is the code box shadow used in the Figma file 

```suggestion
    box-shadow: 0px 15px 50px rgba(22, 29, 63, 0.16);
```

You can get this by clicking on the element, and clicking on the code section in Figma, to see the equivalent CSS as shown in the screenshot below:

<img width="1436" alt="Screenshot 2020-04-18 at 9 53 25 PM" src="https://user-images.githubusercontent.com/26963369/79671049-45eb3300-81bf-11ea-92e2-1d9ec0e0c479.png">